runtime/shared_common/CacheMap.cpp

```diff
@@ -6327,22 +6327,23 @@ SH_CacheMap::matchAotMethod(MethodSpecTable* specTable, IDATA numSpecs, J9UTF8*
 		if (NULL == specTable[i].methodSig) {
 			/* Any signature */
 			sigMatch = true;
-		} else if (NULL == J9UTF8_DATA(romMethodSig)) {
-			/* Signature is missing */
-			sigMatch = false;
 		} else {
-			char* methodSig = (char *)J9UTF8_DATA(romMethodSig);
-			char* sigStart = strchr(methodSig, '(') + 1;
-			char* sigEnd = strchr(methodSig, ')');
-			IDATA sigLength = sigEnd - sigStart;
+			const char *methodSig = (const char *)J9UTF8_DATA(romMethodSig);
+			const char *sigStart = (const char *)memchr(methodSig, '(', J9UTF8_LENGTH(romMethodSig));
+			const char *sigEnd = (const char *)memchr(methodSig, ')', J9UTF8_LENGTH(romMethodSig));
 
-			/* Use string between '(' and ')' only */
-			if (sigLength < 0) {
+			if ((NULL == sigStart) || (NULL == sigEnd) || (sigEnd <= sigStart)) {
 				sigMatch = false;
 			} else {
-				sigMatch = (TRUE == wildcardMatch(specTable[i].methodSigMatchFlag,
-										(const char *)specTable[i].methodSig, specTable[i].methodSigLength,
-										(const char *)sigStart, sigLength));
+				/* Use string between '(' and ')' only. */
+				IDATA sigLength = sigEnd - sigStart - 1;
+
+				sigMatch = (TRUE == wildcardMatch(
+								specTable[i].methodSigMatchFlag,
+								(const char *)specTable[i].methodSig,
+								specTable[i].methodSigLength,
+								sigStart,
+								sigLength));
```

Comment:
When split, the arguments should be indented two levels (not more) than the beginning of the statement being continued.

The example in https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line is less clear than it could be, but it shows the arguments indented eight spaces; elsewhere it says indent only with tabs and that tabs are four spaces.